frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -228,6 +229,29 @@ export class DragElementService {
                     true
                 );
                 break;
+            case 'simulatedRegion': {
+                // This ratio has been determined by trial and error
+                const height = SimulatedRegion.image.height / 23.5;
```

Comment:
In SWT we would say magic number and anyway a single magic number with a comment is a big code smell

Reply:
#374 would hopefully encapsulate this number. At the moment this is some sort of conversion factor from the placeholder image height to the image height on the canvas. I could imagine that this conversion is actually not linear...